contrib/logger/sync_logging.md

```diff
@@ -5,8 +5,8 @@
 The logs can be configured using two functions: 
 
 ```js
-   Realm.App.Sync.setLogLevel()
-   Realm.App.Sync.setLogger()
+Realm.setLogLevel(/*..*/);
+Realm.setLogger(/*..*/);
 ```
 
 These two functions are the entry point to configure the sync behaviour which is handled natively. Then in C++ land, we use the ``logger.hpp`` class to isolate our JS engine (``js_sync.hpp``) from the other two frameworks we depend on which are ``object-store`` and `realm-core`.
```

Comment:
This line seems outdated (as it mentions `js_sync.hpp`. We might just want to delete the entire file, since this should now be controllable through the `syncLogLevel` context variable as outlined here: https://github.com/realm/realm-js/tree/main/integration-tests#setting-context

Also, this might change when / if we merge https://github.com/realm/realm-js/pull/6305

Reply:
You're right, must have missed that last sentence. Deleting the file seems reasonable. Regarding #6305, once that gets merged (or in that same PR), we can just update the context variable 👍